playbook/app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx

```diff
@@ -31,11 +31,17 @@ const FixedConfirmationToast = (props: FixedConfirmationToastProps) => {
   )
   const icon = iconMap[status]
 
+  const handleClick = () => {
+    if (closeable) {
+      toggleToast(false)
+    }
```

Comment:
Here we can simplify to one line by simply inverting the conditional value

```suggestion
    toggleToast(!closeable)
```

Reply:
🤦‍♂️ true